### PR TITLE
Updated engine build.gradle files with latest SDK artifacts

### PR DESCRIPTION
**Pull request by f-odhiambo**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #[issue number] 

**Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt [hunk 1]
```diff
@@ -161,6 +162,10 @@ open class AppMainActivity : BaseMultiLanguageActivity(), OnSyncListener {
       }
       is State.Finished -> {
         showToast(getString(R.string.sync_completed))
+        showToast(getString(R.string.sync_completed))
+        val totalTime = ChronoUnit.SECONDS.between(appMainViewModel.syncStartedAt, state.result.timestamp)
+        //showToast(getString(R.string.sync_completed) + " in "+totalTime + " seconds")
+        Timber.d(getString(R.string.sync_completed) + " in "+totalTime + " seconds")
```
Comment (ellykits) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt:168]: Was this used for debugging?

Comment (ellykits) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt:168]: Can we have this in another PR if this functionality is needed.

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt [hunk 2]
```diff
@@ -100,6 +100,7 @@ constructor(
   val navigationConfiguration: NavigationConfiguration by lazy {
     configurationRegistry.retrieveConfiguration(ConfigType.Navigation)
   }
+  var syncStartedAt = OffsetDateTime.now()
```
Comment (ellykits) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt:103]: Can we have this in another PR if this functionality is needed.

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt (hunk 1 above)
Comment (f-odhiambo) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt]: ok

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt (hunk 2 above)
Comment (f-odhiambo) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt]: ok

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt [hunk 3]
```diff
@@ -161,6 +162,10 @@ open class AppMainActivity : BaseMultiLanguageActivity(), OnSyncListener {
       }
       is State.Finished -> {
         showToast(getString(R.string.sync_completed))
+//        showToast(getString(R.string.sync_completed))
+//        val totalTime = ChronoUnit.SECONDS.between(appMainViewModel.syncStartedAt, state.result.timestamp)
+//        // showToast(getString(R.string.sync_completed) + " in "+totalTime + " seconds")
+//        Timber.d(getString(R.string.sync_completed) + " in " + totalTime + " seconds")
```
Comment (ellykits) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt:168]: Delete commented code

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt [hunk 4]
```diff
@@ -100,6 +100,7 @@ constructor(
   val navigationConfiguration: NavigationConfiguration by lazy {
     configurationRegistry.retrieveConfiguration(ConfigType.Navigation)
   }
+//  var syncStartedAt = OffsetDateTime.now()
```
Comment (ellykits) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt:103]: Delete commented code. Create a new issue for this functionality.

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt (hunk 3 above)
Comment (f-odhiambo) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt]: Done

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt (hunk 4 above)
Comment (f-odhiambo) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt]: Done

**Review by ellykits — APPROVED** [review]

**Pull request merged**